app/profile/routes.py

```diff
@@ -0,0 +1,15 @@
+from flask import Blueprint, request, jsonify
+from flask_jwt_extended import jwt_required
+from app.profile import controllers
+
+user_bp = Blueprint('user', __name__)
+
+@user_bp.route('/auth/profile', methods=['GET'])
```

Comment:
```suggestion
user_bp = Blueprint('user', __name__, url_prefix='/profile')

@user_bp.route('', methods=['GET'])
```

I feel like this makes sense to do since there's a separate profile package now